crates/security/src/lib.rs

```diff
@@ -49,6 +56,9 @@ decl_storage! {
         /// Integer increment-only counter, used to prevent collisions when generating identifiers
         /// for e.g. issue, redeem or replace requests (for OP_RETURN field in Bitcoin).
         Nonce: U256;
+
+        /// like frame_system::block_number, but this one only if the parachain status is RUNNING.
```

Comment:
This could benefit from a longer description, specifically we should mention why it is important that our other modules should check this instead of the system height.